packages/commonwealth/client/scripts/views/components/proposals/voting_results.tsx

```diff
@@ -21,14 +21,31 @@ import {
   YesNoRejectVotingResult,
 } from './voting_result_components';
 import useForceRerender from 'hooks/useForceRerender';
+import { IVote } from 'models/interfaces';
+import Proposal from 'models/Proposal';
+import useAaveProposalVotesQuery from 'state/api/proposals/aave/fetchAaveProposalVotes';
+import { ChainNetwork } from 'common-common/src/types';
+import useAaveProposalsQuery from 'state/api/proposals/aave/fetchAaveProposals';
```

Comment:
some imports are defined but not used, remove them which are useless

Reply:
Removed.